README.md

```diff
@@ -46,27 +46,45 @@ Grasscutter uses Gradle to handle dependencies & building.
 
 **Requirements:**
 
-- [Java SE Development Kits - 17](https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html) or higher
+- [Java Development Kit 17](https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html) or higher
 - [Git](https://git-scm.com/downloads)
+- [NodeJS](https://nodejs.org/en/download) (Optional, for building the handbook)
 
-##### Windows
+##### Clone
 
 ```shell
 git clone --recurse-submodules https://github.com/Grasscutters/Grasscutter.git
 cd Grasscutter
+```
+
+##### If not building the handbook
+
+Add the following line to `gradle.properties`:
+
+`skipHandbook`
+
```

Comment:
this should probably just include the build commands with `-PskipHandbook=1`

example:
```shell
.\gradlew jar -PskipHandbook=1
```